apps/gpc-verifier/app/util/verifyProof.ts

```diff
@@ -32,16 +28,16 @@ export const verifyProof = async (
   // Note that gpcBindConfig might not always pick the same circuit as gpcProve, since it doesn't know
   // the size of the inputs. so here we would like to use the circuitIdentifier returned by gpcProve.
   const vConfig = {
-    ...proofRequestBoundConfig,
-    circuitIdentifier: proofObj.boundConfig.circuitIdentifier
+    ...proofRequest.proofConfig,
```

Comment:
This is no longer going through canonicalization (via binding) which may not be an issue with the config you're using, but seems like a best practice.  I notice you removed the explicit binding step, but expected to show up again later.
Maybe the binding should actually happen on constructing the proof request?  The requestor could ask for a proof with a specific circuit, which isn't a terrible idea if the prover and verifier were ever out of sync about the list of circuits.

Reply:
I got confused somehow. Added the explicit binding step back.